_posts/2018-11-29-craft-consultants.md

```diff
@@ -12,9 +12,9 @@ square_related:
 feature_image: posts/2018-11-29-craft-consultants/lego-superheroes.jpg
 ---
 
-> As aspiring Software Craftsmen we are raising the bar of professional software development by practicing it and helping others learn the craft.
+> As aspiring Software Crafters we are raising the bar of professional software development by practicing it and helping others learn the craft.
 
-[Software Craftsmanship Manifesto](http://manifesto.softwarecraftsmanship.org/)
+[Software Craft Manifesto](http://manifesto.softwarecraftsmanship.org/)*
```

Comment:
This is wrong as the name is `Manifesto for Software Craftsmanship` (I took the wrong original name). And as far as I have learned, it is a title - hence, no changes can be made.

Source: https://www.chicagomanualofstyle.org/home.html